src/Pod.ts

```diff
@@ -401,6 +408,94 @@ export default class Pod {
     return this.memberPods;
   };
 
+  /**
+   * Calls a Pod method via a persona.
+   * The persona object can be fetched by using `Pod.getPersonas(address)`
+   * E.g.,
+   * ```
+   * callAsPersona(
+   *  pod.mintMember,
+   *  [newMember],
+   *  { type: 'admin', address: userAddress },
+   *  signer,
+   * )
+   * ```
+   * The sender must be a signer in the admin case, or the address of the sender.
+   * The sender must be a member of the relevant pod.
+   *
+   * @param method
+   * @param args
+   * @param persona
+   * @param sender
+   * @returns
+   */
+  callAsPersona = async (
+    method: any,
+    args: Array<any>,
+    persona: { type: string; address: string },
+    sender?: ethers.Signer | string,
+  ) => {
+    switch (persona.type) {
+      case 'admin':
+        if (!sender) throw new Error(`Expected sender to be signer, but received ${sender}`);
+        args.push(sender);
```

Comment:
why are we pushing sender to args here?

Reply:
Because admin calls require the signer, check the signature for `mintMember` for example to see how that works.